quantii/src/lib.rs

```diff
@@ -70,9 +74,31 @@ impl ardaku::System for System {
 pub fn setup() -> ! {
     use ardaku::System;
 
-    System.write(b"\n\n=== ARDAKU STARTED ===\n");
+    System.write(b"Quantii OS v0.0.1");
+
+    System.write(b"\n=== ARDAKU STARTED ===\n");
+
+    let mut msg: String;
+    match ardaku::start(System, APP_EXE) {
+        Ok(_) => msg = "\n=== ARDAKU SUCCESSFULLY EXECUTED ===\n".to_owned(),
+        Err(e) => match e {
+            ArdakuError::InvalidWasm => {
+                msg = "Ardaku: Error: Invalid WASM file".to_owned()
```

Comment:
Should combine let and match into one expression.

```suggestion
    let msg = match ardaku::start(System, APP_EXE) {
        Ok(_) => "\n=== ARDAKU SUCCESSFULLY EXECUTED ===\n",
        Err(e) => match e {
            ArdakuError::InvalidWasm => {
                "Ardaku: Error: Invalid WASM file"
```